ow_plexil/src/plans/TestArmMoveJoints.plp

```diff
@@ -0,0 +1,26 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// A test plan for the ROS action ArmMoveJoints
+// that commands the 6 arm joints both sequentially and in parallel
+
+#include "plan-interface.h"
+
+TestArmMoveJoints:
+{
+  // Three sets of angles for each joint
+  Real angles1[6] = #(-1.0 0.5 0.0 0.03 0.0 0.0);
+  Real angles2[6] = #(0.0 0.5 0.0 0.0 0.0 0.0);
+  Real angles3[6] = #(0.0 1.0 0.0 0.0 0.0 0.0);
+
+  log_info ("Starting TestArmMoveJoints plan...");
+
+  LibraryCall ArmMoveJoints (Relative=false, Angles=angles1);
+
```

Comment:
This call produced an error in the simulator:

`ValueError: operands could not be broadcast together with shapes (6,) (0,) `

This needs some debugging, but I suspect that the vectors you define here are not getting correctly converted into the vectors that the Python-based action client gets (this error comes from Python).  I can execute these three commands manually using the Python client scripts without any problems.

Try inspecting (e.g. printing out) the array that gets down to `OwInterface::armMoveJoints` and see if you can find the source of the problem.   If the array looks good there, have a look at what's coming into `ow_simulator/ow_lander/scripts/arm_action_servers.py`